src/components/menu/menu-gestures.ts

```diff
@@ -57,14 +57,14 @@ export class MenuContentGesture extends SlideEdgeGesture {
   }
 
   onSlide(slide: SlideData, ev: any) {
-    const z = (this.menu.isRightSide ? slide.min : slide.max);
+    const z = ((this.menu.isRightSide && !this.plt.isRTL) || (!this.menu.isRightSide && this.plt.isRTL) ? slide.min : slide.max);
```

Comment:
`(this.menu.isRightSide && !this.plt.isRTL) || (!this.menu.isRightSide && this.plt.isRTL)`
is logically equal to:
`this.menu.isRightSide !== this.plt.isRTL`

Can you fix this in all of the expressions in this PR?